app/models/game_lineup_slot.rb

```diff
@@ -0,0 +1,18 @@
+class GameLineupSlot < ActiveRecord::Base
```

Comment:
Everything in this class needs a corresponding spec line, minus the relationships which look like they're already covered.